### PR TITLE
New abt subtitle

### PR DESCRIPTION
**Pull request by lvu063**
I changed subtitles on the About page from
- About Us to WHO WE ARE
- Our Core Values to WHAT WE STAND FOR
- Our Story to HOW WE STARTED

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (patfire12345) [inline .vscode/settings.json:1]: Delete this please.

**Review by patfire12345 — CHANGES_REQUESTED** [review]
Delete the vscode file, otherwise LGTM!

File: .vscode/settings.json (hunk 1 above)
Comment (lvu063) [reply .vscode/settings.json]: I resolved it.

**Review by patfire12345 — APPROVED** [review]
LGTM!

**Pull request merged**